content/reference/cd-github.md

```diff
@@ -5,7 +5,6 @@ expanded_url: /reference/service/
 menu:
   reference:
     parent: teams
-    weight: 4
```

Comment:
This line in should remain in this file. I know it's confusing as this file is prefixed with `cd-*` like all the other files, but this one is actually part of the "Teams & Collaboration" menu, not "Continuous Delivery" like all the others. Sorry for not being more clear about that.

```suggestion
    weight: 4
```

Reply:
No worries. I re-added the line (and removed the Codefresh alias)